benefits/templates/admin/agency-base.html

```diff
@@ -26,6 +26,31 @@ <h1 class="p-0 m-0 text-white">Administrator</h1>
 {% endblock branding %}
 
 {% block usertools %}
+    {% if has_permission %}
+        <div id="user-tools">
+            {% block welcome-msg %}
+                <span class="text-uppercase text-white">
+                    {% translate "Welcome," %}
```

Comment:
To be consistent with how copy is maintained in our admin / in-person templates, I suggest not using `translate` at all in this template.

I realize you probably just grabbed this from the original Django template and might be thinking of our previous discussion about preserving original functionality if it doesn't hurt anything. But in this case with copy, we've said we don't need to worry about translation for now.

Reply:
Ah that's right, thanks for reminding me about that 👍 I made the change in aa73dca7d3981c5bd06504cb9aa5d23011dfc39e.